bw2io/strategies/simapro.py

```diff
@@ -15,6 +15,7 @@
 import re
 import numpy as np
 from stats_arrays import LognormalUncertainty
+import bw2parameters
```

Comment:
This means we have to add bw2parameters to the setup requirements. What do you say @cmutel?